mkinitcpio-install.sh

```diff
@@ -52,6 +52,35 @@ run_command() {
     esac
 }
 
+concatenate_unit() {
+    # concatenate units and their potential drop-in files
+    # we do this manually, as in a chroot `systemctl cat` will not work
+    local unit_name="$1"
+    local unit_content=""
+    local service_path=""
+    local override=""
+    # add the top-most service
+    for service_path in {/usr/lib,/etc,/run}/systemd/system; do
```

Comment:
I don't think runtime changes should be considered since they are always overriden by /etc drop-ins.

Reply:
Only the first service file should be taken into account since it's a replacement and not an override (like drop-ins).
Since runtime should not be taken into account, the order is `/etc`, `/usr/lib`.

Reply:
Fair enough for the runtime changes.

As the ordering is `/etc`, `/usr/lib` the shell expansion is `for service_path in {/usr/lib,/etc}/systemd/system; do` though (the last one wins in the above implementation).

Reply:
Yea, I wrote it when I already had my next comment in mind (print and break asap).